setup.cfg

```diff
@@ -37,6 +37,6 @@ omit =
 norecursedirs = .git .idea .cache
 
 [flake8]
-ignore = E124,E125,E402,W504
+ignore = E124,E125,E402,W504,A005
```

Comment:
that should not be in my PR, that is already in develop, how can I fix that?

Reply:
@joernu76 @matrss do you have an idea, what I have to do?

Reply:
How did you add the commit here? I am also surprised it shows up in the diff for all commits...

You can drop the commit from your branch and either merge in develop, or rebase your branch on top of develop. Those are the two ways I would generally update a PR branch.